features/component_generator/stimulus.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Component generator - Stimulus
+  Scenario: Component with namespaces and stimulus
+    Given I run `rails new my_app --skip-spring`
+    And I cd to "my_app"
+    And I append to "Gemfile" with:
+    """
+        gem 'webpacker', '~> 3.0'
```

Comment:
@andrzejsliwa You can omit this line now. Handled by `.gemspec`.

Reply:
done